tests/build_arm.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -x
+
+ALPINE_VER="3.8"
+DISTRO="balenalib/rpi-alpine:$ALPINE_VER"
+# Used for Radicale
+EDGE_DISTRO="balenalib/rpi-alpine:edge"
+# Used for webmails
```

Comment:
Edge is no longer used:
https://github.com/Mailu/Mailu/blob/13d6cf2fdbcf9b4e3bd31d5f8db467ee340cbc74/optional/radicale/Dockerfile#L1